ark/type/parser/objectLiteral.ts

```diff
@@ -1,12 +1,16 @@
 import { schema, type Inner } from "@arktype/schema"
+import { keywords } from "@arktype/schema/internal/keywords/keywords.js"
```

Comment:
Change this to `@arktype/schema` (I know there's some obnoxious stuff that happens with auto imports that I can't resolve)